src/ScenarioManagerTreeList/components/ScenarioNode/ScenarioNode.js

```diff
@@ -35,11 +48,29 @@ const ScenarioNode = ({
 
   return (
     <React.Fragment>
+      <ConfirmDeleteDialog
```

Comment:
Could be interesting to re-use `SimpleTwoActionsDialog` component?
It should match 90% of your need (the other 10% could be an upgrade of this component :p )

Reply:
I'm not a huge fan of reusing the _SimpleTwoActionsDialog_ component for all dialogs with 2 options, because this component should have too many options to be able to customize its content (typography variant, translation keys, translation default values), its behavior (enable or disable backdrop click to cancel the dialog) and its metadata (aria-labelledby, aria-describedby, ids, data-cy)